phpstan-baseline.neon

```diff
@@ -392,6 +392,14 @@ parameters:
 			count: 1
 			path: src/Testing/PHPStanTestCase.php
 
+		-
```

Comment:
I was able to drop all uses of `isClassString` and `instanceof ClassStringType` with [48f8635](https://github.com/phpstan/phpstan-src/pull/1970/commits/48f863561c9dcfa359a51ba8ee395970eab8a050) but these 2 occurences. These seemed harder to solve, therefore I put them into the baseline.

Reply:
I might be missing something, but it looks like the internals of `isClassString()` should be moved into `isClassStringType()` and wrapped in a trinary. At least you don't have to use the deprecated method in the new one then. You could even turn it around and let the deprecated one use the new one internally?

Not sure what the problem with the other usage was 🤔

Reply:
after a couple of hours sleep I was able to resolve the remining stuff :)